### PR TITLE
New Package: alure

### PR DESCRIPTION
**Pull request by Magister-Bergerac**

### CODE REVIEW

File: srcpkgs/alure/template [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+pkgname=alure
+version=1.2
+revision=1
+build_style=cmake
+makedepends="libopenal-devel"
+short_desc="Utility library to help manage common tasks with OpenAL applications. "
+maintainer="Simon Whelan <si.whelan@gmail.com>"
+license="GPL-3"
+homepage="http://kcat.strangesoft.net/alure.html"
+distfiles="http://kcat.strangesoft.net/alure-releases/alure-1.2.tar.bz2"
```
Comment (maxice8) [inline srcpkgs/alure/template]: use ${version} instead of hardcoding

**Comment by cr6git** [comment]
`license="GPL-3"`

Not sure about that...
The Arch [package](https://www.archlinux.org/packages/community/x86_64/alure/) is "MIT", while the Debian [copyright](http://metadata.ftp-master.debian.org/changelogs/main/a/alure/alure_1.2-6_copyright) says "Expat".

File: srcpkgs/alure/template [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Template file for 'alure'
+pkgname=alure
+version=1.2
+revision=1
+build_style=cmake
+makedepends="libopenal-devel"
+short_desc="Utility library to help manage common tasks with OpenAL applications. "
+maintainer="Simon Whelan <si.whelan@gmail.com>"
+license="GPL-3"
```
Comment (maxice8) [inline srcpkgs/alure/template]: change to MIT and add

```
post_install() {
    vlicense COPYING
}
```

at the end

File: srcpkgs/alure/template [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Template file for 'alure'
+pkgname=alure
+version=1.2
+revision=1
+build_style=cmake
+makedepends="libopenal-devel"
+short_desc="Utility library to help manage common tasks with OpenAL applications. "
```
Comment (jnbr) [inline srcpkgs/alure/template]: remove trailing white space and dot

File: srcpkgs/alure/template [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+maintainer="Simon Whelan <si.whelan@gmail.com>"
+license="X11/MIT"
```
Comment (maxice8) [inline srcpkgs/alure/template]: we have no such concept of X11/MIT, we use MIT for all licenses.

**Comment by Vaelatern** [comment]
single commit.

**Pull request closed without merging**